src/enums/MeldEnum.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Meld enum
+ */
+export enum MeldEnum {
+  TRIPLET = 'TRIPLET',
+  CONSECUTIVE = 'CONSECUTIVE',
+  QUAD = 'QUAD',
+}
```

Comment:
Possibly add WIN = 'WIN' enum as well

Reply:
add `WIN = 'WIN'`  to `MeldEnum`?`WIN` could be a state for a round, not a meldType right?

Reply:
If a player can win, WIN takes the highest priority when deciding played tile interaction

Reply:
kk